service/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Set the base/parent image
+# see https://github.com/broadinstitute/dsp-appsec-blessed-images/tree/main/jre
+# Google's distroless images are openjdk, this is the simplest with adoptopenjdk
+FROM us.gcr.io/broad-dsp-gcr-public/base/jre:17-debian
```

Comment:
I'm wondering about this comment. Is it true that we shouldn't be using distroless since it uses openjdk? I don't think that's true for the JDK 17 images at least. Catalog and BPM are using distroless `us.gcr.io/broad-dsp-gcr-public/base/jre:17-distroless`